### PR TITLE
add identity docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

Original PRs:

- https://github.com/moonbeam-foundation/moonbeam-docs/pull/748/
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/805/

### CODE REVIEW

File: builders/pallets-precompiles/precompiles/identity.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+Identity Precompile是一个Solidity接口，用于创建、管理和检索链上身份信息。身份与账户相关联，并包括个人信息，例如法定姓名、对外显示的名称、网站、Twitter名称、Riot（现为Element）名称等。您也可以通过填写自定义字段包含任何其他相关信息。
+
+The Identity Precompile interacts directly with Substrate's [Identity Pallet](/builders/pallets-precompiles/pallets/identity){target=_blank} to provide the functionality needed to create and manage identities. This pallet is coded in Rust and is normally not accessible from the Ethereum side of Moonbeam. However, the Identity Precompile allows you to access this functionality directly from the Solidity interface.
+
+Identity Precompile直接与Substrate的[Identity Pallet](/builders/pallets-precompiles/pallets/identity){target=_blank}交互，以提供创建和管理身份所需的功能。此Pallet以Rust编写，通常无法从Moonbeam的以太坊侧直接访问。然而，Identity Precompile允许您直接从Solidity接口访问此功能。
+
+The Identity Precompile is only available on Moonbase Alpha and is located at the following address:
+
+Identity Precompile位于以下地址：
```
Comment (wuzhong-papermoon) [inline builders/pallets-precompiles/precompiles/identity.md:21]:
```suggestion
Identity Precompile只在Moonbase Alpha可用，它位于以下地址：
```

[the remark this answers is not in the document]
Comment (wuzhong-papermoon) [reply builders/pallets-precompiles/precompiles/identity.md:21]: Good catch.

File: builders/pallets-precompiles/precompiles/identity.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    --8<-- 'code/builders/pallets-precompiles/precompiles/identity/abi.js'
+    ```
+
+Once you have the ABI, you can interact with the precompile using the Ethereum library of your choice. Generally speaking, you'll take the following steps:
+
+有了ABI后，您可以根据您的选择使用以太坊库与预编译交互。一般情况下，您需要执行以下步骤：
+
+1. Create a provider
+
+   创建提供商
```
Comment (wuzhong-papermoon) [inline builders/pallets-precompiles/precompiles/identity.md:151]:
```suggestion
   创建 provider
```

**Comment by eshaben** [comment]
@wuzhong-papermoon can you please approve this PR?

**Review by wuzhong-papermoon — APPROVED** [review]

**Pull request merged**